src/cla/dummy.rs

```diff
@@ -10,6 +10,9 @@ impl DummyConvergenceLayer {
     pub fn new() -> DummyConvergenceLayer {
         DummyConvergenceLayer {}
     }
+    pub const fn ext_help_str() -> &'static str {
```

Comment:
a new `HelpStr` trait is required because ConvergenceLayerAgent is used by `enum_dispatch` which requires a `self` to match on. since this function does not depend on an object, embedding in the same trait is not possible